integration_tests/models/dummy_model.sql

```diff
@@ -0,0 +1,4 @@
+{{ config(materialized='view') }}
+    begin;
+    commit;
```

Comment:
I think the syntax for transactions is slightly different between DBs + Databricks actually don't support them at all (interesting to see what happens in the CI once it passes)

In any case may be safer to do something like `SELECT 1` that will always work